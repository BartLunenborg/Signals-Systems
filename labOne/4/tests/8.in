10000 100 100 73 1.57
