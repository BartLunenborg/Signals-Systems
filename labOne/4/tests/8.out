[0,-4,-9,-13,-18,-22,-27,-31,-35,-40,-44,-48,-52,-56,-59,-63,-66,-70,-73,-76,-79,-82,-84,-86,-89,-91,-92,-94,-95,-97,-98,-98,-99,-99,-99,-99,-99,-99,-98,-97,-96,-95,-93,-92,-90,-88,-85,-83,-80,-78,-75,-71,-68,-65,-61,-58,-54,-50,-46,-42,-38,-33,-29,-25,-20,-16,-11,-6,-2,2,6,11,15,20,24,29,33,37,42,46,50,54,57,61,65,68,71,74,77,80,83,85,88,90,92,93,95,96,97,98]
